### PR TITLE
Prevent HTML injection in the alerts pane

### PR DESCRIPTION
**Pull request by psiinon**
Related to #3234 but affecting JLabels instead of the tree

### CODE REVIEW

File: src/org/zaproxy/zap/utils/ZapLabel.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import javax.swing.JTextArea;
+import javax.swing.UIManager;
+
+/**
+ * An alternative to JLabel which disables HTML and supports wrapping.
+ * Use this class instead of JLabel where the text could include HTML that you
+ * do not want to render.
+ * @author psiinon
+ *
```
Comment (thc202) [inline src/org/zaproxy/zap/utils/ZapLabel.java]: Could you add the since tag?

File: src/org/zaproxy/zap/utils/ZapLabel.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+ */
+public class ZapLabel extends JTextArea {
+
+	private static final long serialVersionUID = 1L;
+
+	public ZapLabel() {
+		this.setEditable(false);
```
Comment (thc202) [inline src/org/zaproxy/zap/utils/ZapLabel.java]: Could be removed (already set in `setDefaults`)?

File: src/org/zaproxy/zap/utils/ZapLabel.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		setDefaults();
+	}
+
+	public ZapLabel(String text) {
+		super(text);
+		setDefaults();
+	}
+
+	private void setDefaults() {
+		this.putClientProperty("html.disable", Boolean.TRUE);
```
Comment (thc202) [inline src/org/zaproxy/zap/utils/ZapLabel.java]: This could be removed (not needed for this UI component).

Comment (psiinon) [inline src/org/zaproxy/zap/utils/ZapLabel.java]: Updated to fix those issues

**Comment by thc202** [comment]
Thanks!

**Pull request merged**